source/tools/testConsistency/ClientApp/app.test.js

```diff
@@ -0,0 +1,113 @@
+// To run these tests, first ensure that the flask app for nodeJs is running at 127.0.0.1 port 5000
```

Comment:
if possible, it would be nice if this was configurable on the command line

Reply:
added a bat file in the client app folder that does all this work for us